app/queries/get_embedded_content.rb

```diff
@@ -0,0 +1,39 @@
+module Queries
+  class GetEmbeddedContent
+    attr_reader(:target_content_id)
+
+    def initialize(target_content_id)
+      self.target_content_id = target_content_id
+    end
+
+    def call
+      if Document.find_by(content_id: target_content_id).nil?
+        message = "Could not find an edition to get embedded content for"
+        raise CommandError.new(code: 404, message:)
+      end
+
+      Presenters::Queries::EmbeddedContentPresenter.new(
+        target_content_id,
+        host_editions,
+      ).present
+    end
+
+  private
+
+    def host_editions
+      @host_editions ||= Edition.live
+        .joins(:links)
+        .joins("LEFT JOIN links AS primary_links ON primary_links.edition_id = editions.id AND primary_links.link_type = 'primary_publishing_organisation'")
+        .joins("LEFT JOIN documents ON documents.content_id = primary_links.target_content_id")
+        .joins("LEFT JOIN editions AS org_editions ON org_editions.document_id = documents.id")
+        .where(links: { target_content_id: }) # TODO: Inlcude `link_type: "embedded"`` when it is supported.
```

Comment:
I believe it is supported, only it's called "embed", rather than "embedded" (added in 372952b1efb68b3d8f78b855d9aa3202a46fd488). Unless I misunderstand?

Reply:
Oh super, I didn't realise this had happened already. Thank you 🙌